internal/msalbase/account_test.go

```diff
@@ -8,17 +8,19 @@ import (
 	"testing"
 )
 
-var accHID = "hid"
-var accEnv = "env"
-var accRealm = "realm"
-var authType = "MSSTS"
-var accLid = "lid"
-var accUser = "user"
+var (
+	accHID   = "hid"
+	accEnv   = "env"
+	accRealm = "realm"
+	authType = "MSSTS"
+	accLid   = "lid"
```

Comment:
>accLid [](start = 1, length = 6)

nit: is this local account id? same comment as for homeAccountId, to make it easier to read.

Reply:
Same as above, this is unchanged from the original code and probably wont' survive long term.